src/components/Sidebar/components/NavItem/NavItem.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+
+const baseStyles = ({ theme }) => css`
+  label: nav-item;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  height: auto;
+  margin: ${theme.spacings.mega};
+  padding: ${theme.spacings.bit};
+  cursor: pointer;
+  color: ${theme.colors.n500};
+  fill: ${theme.colors.n500};
+`;
+
+const hoverStyles = ({ theme, selected }) =>
+  !selected &&
+  css`
+    label: nav-item__hover;
```

Comment:
Just a tiny hint on the BEM-style notation. We use the `__` to separate elements from the block and `--` to separate the modifier from the element. So, it's `B__E--M`. `hover` and `active` are modifiers.